packages/vscode-extension/README.md

```diff
@@ -1,64 +1,85 @@
 
-The following types of SDF projects are available:
-- SuiteApp projects — self-contained, standalone projects that you can publish in the SuiteApp Marketplace for other users to download and install.
-  > 💡 To read more about SuiteApp projects, visit [NetSuite's Help Center.](https://system.netsuite.com/app/help/helpcenter.nl?fid=subsect_1509931104.html)
-- Account customization projects — intended for customizations on accounts you own, such as your production, development, and sandbox accounts.
-  > 💡 To read more about account customization projects, visit [NetSuite's Help Center.](https://system.netsuite.com/app/help/helpcenter.nl?fid=subsect_1510680449.html)
+## <font size="6"> <span style="color:gray"> Installation prerequisites </font> </span>
+The following software is required to work with SuiteCloud Extension for Visual Studio Code:
+- Node.js version 14.16.0 LTS
+- Oracle JDK version 11
 
-![Account customization project](resources/acpMain.png)
+Read the detailed list of prerequisites in <a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_159223197655.html" rel="noopener noreferrer" target=blank>Installation Requirements for SuiteCloud Extension for Visual Studio Code</a>.
 
-SuiteCloud extension for Visual Studio Code is part of SuiteCloud Software Development Kit (SuiteCloud SDK), a set of tools to customize NetSuite accounts.
+## <font size="6"> <span style="color:gray"> Installing SuiteCloud Extension for Visual Studio Code </font> </span>
+To install SuiteCloud Extension for Visual Studio Code, follow these steps:
 
-💡 To read more about SuiteCloud SDK, visit [NetSuite's Help Center.](https://system.netsuite.com/app/help/helpcenter.nl?fid=chapter_156026236161.html)
+1. Launch Visual Studio Code and select the **Extensions** tab.
+2. In the search field, enter **SuiteCloud Extension for Visual Studio Code**.
+3. Click **Install**.
+4. Restart Visual Studio Code after installation.
 
-## Installation prerequisites
-The following software is required to work with SuiteCloud extension for Visual Studio Code:
-- Node.js version 14.16.0 LTS
-- Oracle JDK version 11
+See more details about installation in <a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_159223197655.html" rel="noopener noreferrer" target=blank>Installing and Setting Up SuiteCloud Extension for Visual Studio Code</a>.
+
+## <font size="6"> <span style="color:gray"> Getting Started </font> </span>
+
+### <font size="5"> <span style="color:gray"> How to Trigger the Command Palette </font>
+Interact through the Command Palette to use the SuiteCloud Extension for Visual Studio Code. The Command Palette allows you to perform several actions in a straightforward way, improving the user experience to the most. To trigger it, use the shortcut, which is Ctrl+Shift+P by default, or click **View** > **Command Palette**. 
```

Comment:
Do we need to say these ?   'straightforward way'  and   'improving the user experience to the most'
Seems unnecessary to me.

Reply:
Deleted. It was added to highlight the use of the Command Palette, but I think it didn't add much.